src/main/java/bio/terra/cli/command/resource/ListTree.java

```diff
@@ -24,7 +25,6 @@ public class ListTree extends WsmBaseCommand {
   private static final HashMap<UUID, String> ID_TO_NAME = new HashMap<>();
   private static final HashMap<UUID, Boolean> IS_FOLDER = new HashMap<>();
   private static final UUID ROOT = UUID.randomUUID();
-  private static final String TERRA_FOLDER_ID_PROPERTY_KEY = "terra-folder-id";
```

Comment:
i don't see the value of changing this to a enum yet. when there are 3+ resource properties that we want to iterate through yes, but given there's only one and we are specifically looking for it, static field is enough.

Reply:
Sg.